pkg/idp/util.go

```diff
@@ -43,3 +46,23 @@ func GetDexSecretConfig(namespace string) v1alpha1.AppExtraConfig {
 		Namespace: namespace,
 		Priority:  25}
 }
+
+func getConnectorsFromSecret(secret *corev1.Secret) (map[string]dex.Connector, error) {
```

Comment:
This function creates a map out of the connector secret so that connectors can be compared more easily